eclipse/che
# Move Happy path PR check pipelines to new CRW CCI

### What does this PR do?
It moves Happy path PR check pipelines to new CRW CCI

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-1495

### How to test this PR?
Trigger:
- Eclipse Che repo Happy path PR check
- Che-Theia repo Happy path PR check
- Che-Dashboard Happy path PR check


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
